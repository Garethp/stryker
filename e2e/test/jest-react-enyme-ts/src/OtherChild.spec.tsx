import React from 'react';
import { shallow } from 'enzyme';
import { OtherChild } from './OtherChild';

describe('OtherChild Component', () => {
  it('should render correctly', () => {
    expect(shallow(<OtherChild />)).toMatchSnapshot();
  });
});
