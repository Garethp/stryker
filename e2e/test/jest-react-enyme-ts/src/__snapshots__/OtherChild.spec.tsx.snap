// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtherChild Component should render correctly 1`] = `<div />`;
